stdlib/exception.gr

```diff
@@ -1,11 +1,14 @@
 /**
  * Utilities for working with the Exception type.
  *
- * The Exception type represents an error that has occured during computation.
+ * The Exception type represents an error that has occurred during computation.
  *
  * @example include "exception"
  *
- * @since 0.3.0
+ * @example exception ExampleError(Number)
+ * @example exception ExampleError
```

Comment:
Were these two lines a mistake

Reply:
No I started adding an example of how to use the data type to the top of the module. I think it's a good place to provide the syntax and maybe in the future it could be replaced with a link to documentation on the actual data type.